README.md

```diff
@@ -137,6 +137,13 @@ from fbtftp.base_server import BaseServer
 
 import os
 
+ip = '10.10.20.75'
```

Comment:
why are you hard coding this ip? there is no need. if you pass `ip=''` below the server will bind to all IPs on the machine where it's running.

Reply:
also if you want a better example you could just have the example use sys.argv or argparse module :)